app/pages/search/search-controls/SearchControls.js

```diff
@@ -1,30 +1,59 @@
+import FontAwesome from 'react-fontawesome';
 import { decamelizeKeys } from 'humps';
 import isEqual from 'lodash/isEqual';
+import omit from 'lodash/omit';
 import queryString from 'query-string';
 import React, { Component, PropTypes } from 'react';
 import Button from 'react-bootstrap/lib/Button';
 import Checkbox from 'react-bootstrap/lib/Checkbox';
 import ControlLabel from 'react-bootstrap/lib/ControlLabel';
 import FormControl from 'react-bootstrap/lib/FormControl';
 import FormGroup from 'react-bootstrap/lib/FormGroup';
+import Row from 'react-bootstrap/lib/Row';
+import Col from 'react-bootstrap/lib/Col';
 import { browserHistory } from 'react-router';
 
 import DatePicker from 'shared/date-picker';
+import LabelSelect from 'shared/form-fields/label-select/LabelSelect';
+
+function getUnitOption(id, label) {
+  return <option key={id} value={id}>{label}</option>;
+}
+
+function renderUnitOptions(units) {
+  const defaultOption = getUnitOption('', 'Kaikki kiinteistöt');
+  const options = Object.keys(units).map((id) => {
```

Comment:
- [ ] Is the order of the keys returned by Object.keys defined? I think it's arbitrary order, so it'd be good to sort these for example alphabetically. Same with the other uses of Object.keys in this file.